README.md

```diff
@@ -56,43 +48,45 @@ guide you on how to create your rados block device.
 
 ## Deploying
 
-Then, to launch the deployment :
+Then, to launch the deployment:
 ```
-./kolla-g5k.py
+ham deploy
 ```
 
 > by default the script use the file `reservation.yaml` as configuration file
 
-This will run all the following phases :
+This will run all the following phases:
 
-* `prepare-node` : reserve physical machine on Grid'5000, deploys the base
-linux distribution, instruments nodes with the monitoring stack, installs Kolla
-prerequisites (runtime dependencies, input files.
+* `up`: reserve physical machine on Grid'5000, deploys the base linux
+  distribution, instruments nodes with the monitoring stack, installs
+  Kolla prerequisites (runtime dependencies, input files).
 
-* `install-os` : launch the Kolla deployment
+* `os`: launch the OpenStack deployment with Kolla
 
-* `init-os` : bootstrap the freshly deployed OpenStack with users, images...
+* `init`: bootstrap the freshly deployed OpenStack with users,
+  images...
 
 
-> Run `./kolla-g5k.py --help` for a full list of command-line arguments.
+> Run `./ham.py --help` for a full list of command-line arguments.
 
-After the execution, the `./current` directory will contain some generated
-files (please note that `current` is a symbolic link toward the real directory
-associated to your deployment)  and the results of the benchmarks
+After the execution, the `./current` directory will contain some
```

Comment:
I've the feeling that when using pip this directory will not be placed in the `.` but will try to be written where the sources reside